src/Microsoft.Azure.SignalR.AspNet/ServiceOptions.cs

```diff
@@ -38,6 +38,12 @@ public class ServiceOptions : IServiceEndpointOptions
         /// </summary>
         public Func<IOwinContext, IEnumerable<Claim>> ClaimsProvider { get; set; } = null;
 
+        /// <summary>
+        /// Gets or sets the func to set tracking client filter from <see cref="IOwinContext" />.
+        /// The clients will be regarded as tracking client only if the function returns true.
+        /// </summary>
+        public Func<IOwinContext, bool> TrackingClientFilter { get; set; } = null;
```

Comment:
naming: `ConfigureDiagnosticClients`?

Reply:
Looks like `Filter` give a condition to the input, something like https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter,
`Configure` pass in an `Action`, somthing like https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-3.1#add-providers.
So I prefer to use `xxxFilter`. does it make sense?

Reply:
`TrackingClient` this is my main concern, it sounds too big a term while actually it is only about logging.

Reply:
So `DiagnosticClientFilter`?

Reply:
DiagnosticClientEnableCondition? Too long?

Reply:
DiagnosticClientTracker ?

Reply:
ClientLoggingFilter?